packages/terra-list/src/ListItem.scss

```diff
@@ -1,30 +1,23 @@
 @import '~terra-mixins';
 @import './variables';
 
-.terra-ListItem-hasChevron {
-  display: block;
-  position: relative; // relative position so the chevron can use absolute position to vertically center itself
+.terra-ListItem {
+  display: table;
+  outline: none;
+  width: 100%;
+}
 
-  [dir='ltr'] & {
-    @include terra-padding-end(20px);
-  }
+.terra-ListItem > * {
+  display: table-cell;
```

Comment:
+1 I'll switch this out to use the arrange component when the chevron is present